### ISSUE TITLE
How to train my  own dataset?

### ISSUE THREAD

**Issue by NARRZ** [lucastabelini/PolyLaneNet#55]
Hello, I am very interested in your research results. My own lane line data set has been converted to tusimple style. How can I use my own data set for training? Thank you!

**Comment by lucastabelini** [comment]
Thank you for your interest. You might also be interested in my latest lane detection work: https://github.com/lucastabelini/LaneATT

If your dataset is in TuSimple's format, you can train a model on it using the same procedure described on the README.md: https://github.com/lucastabelini/PolyLaneNet#training

One of the settings that you will have to change is the root of the dataset (which points to `"datasets/root"` by default). You will also want to modify the number of epochs and the learning rate scheduler.

**Comment by NARRZ** [comment]
Thank you for your reply. I found that I did not really convert my dataset to tusimple format, and I did not find the correct conversion method, do you have any suggestions to convert the data set to tusimple format? Thank you!

**Comment by lucastabelini** [comment]
See #5.
